src/components/Editor.js

```diff
@@ -150,6 +152,30 @@ const Editor = React.createClass({
     }
   },
 
+  toggleBreakpointDisabledStatus(line) {
```

Comment:
I think this method is fine how it is. The logic is cleanly separated and should make tracking things down easier later.